main.rb

```diff
@@ -0,0 +1,47 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
- [ ] Please kindly fix your linters instead of disabling them, fix linters help to write a clean and professional codebase 😁 😉

Reply:
Thanks!
I removed the disabilities of linters!